### DISCUSSION TITLE
Welcome to the McFACTS discussion board!

### DISCUSSION THREAD

**Discussion by JakePost** [bmckernan/test_mcfacts#241] [Announcements]
## 👋 Welcome!
  We’re using GitHub Discussions as a place to connect with other members of our community and provide support for McFACTS. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

We are still in an early development phase, and still have a lot of things we want to implement. So we kindly ask for your patience!

On the development side of things, you are more than welcome to dive into the code and start hacking away! However, please know that we will not be accepting or reviewing large/feature pull requests from outside collaborators until we've hit certain development milestones. This doesn't mean we will completely ignore PRs in this development phase -- If you've got a bug fix, please let us know! -- we are just being **extra** cautious with these first few steps in an attempt to prevent feature creep, while building towards our vision of an _super-ultra-modern-hyper-modular_ codebase. Stay tuned for updates regarding an overview of future features and a project roadmap.

On the application / results side of things: If you are having trouble with any part of McFACTS, reach out! We will keep an eye on the discussion board to provide support where we can, but we also have office hours available where we are more than happy to help debug things.

Visit our [website page](https://saavikford.wixsite.com/saavik/general-7) to see our office hours schedule. Send a message to the email on that page to request the Zoom room password, and to let us know when you'll be joining.

Cheers!

**Comment by bmckernan** [comment]
Welcome!
